src/include/wiredtiger.in

```diff
@@ -3847,12 +3847,6 @@ int wiredtiger_config_validate(WT_SESSION *session,
     WT_EVENT_HANDLER *event_handler, const char *name, const char *config)
     WT_ATTRIBUTE_LIBRARY_VISIBLE;
 
```

Comment:
Had to force remove this from wiredtiger.in (this function only appears in config_api.c and not api_data.py which regenerates wiredtiger.in)  - even if the file regenerates this change remains. This indicates to me there's a section in wiredtiger.in that allows for manual changes to be let in.

Reply:
I suspect you're right - although I'm not sure whether/how it was generated in the first place.

Reply:
I'll ask @luke-pearson if he remembers adding this manually, it was a while ago though so hard to chase.

Reply:
@luke-pearson has confirmed this section is manual and only part of wiredtiger.in is autogenerated :)

Reply:
Great, thanks for following this up! Happy to resolve the thread if you want.